src/Anthropic.Client.Tests/Services/Messages/MessageServiceTest.cs

```diff
@@ -38,6 +42,213 @@ public async Task CreateStreaming_Works()
         }
     }
 
+    [Fact]
+    public async Task CreateStreamingAggregation_WorksNoContent_RawMessageStartEvent()
+    {
+        // arrange
+        var messagesServiceMock = new Mock<IMessageService>();
+        static async IAsyncEnumerable<RawMessageStreamEvent> GetTestValues()
+        {
+            yield return new(new RawMessageStartEvent(GenerateStartMessage()));
+            yield return new(new RawMessageStopEvent());
+            await Task.CompletedTask;
+        }
+        // act
+        messagesServiceMock.Setup(e => e.CreateStreaming(It.IsAny<MessageCreateParams>())).Returns(GetTestValues);

```

Comment:
I think mock setup is typically considered part of the arrange step, so should probably move it there for each test